indico/modules/events/reminders/forms.py

```diff
@@ -56,6 +56,9 @@ class ReminderForm(IndicoForm):
     message = TextAreaField(_('Note'), description=_('A custom message to include in the email.'))
     include_summary = BooleanField(_('Include agenda'),
                                    description=_("Includes a simple text version of the event's agenda in the email."))
+    include_description = BooleanField(_('Include description'),
+                                       description=_("Includes a simple text version of the event's "
```

Comment:
I think "a simple text version of " is not needed here; in case of the agenda we have it to make it clear we don't include a very fancy timetable

Reply:
done